src/NuGet.Core/NuGet.Build.Tasks/LoggingQueue.cs

```diff
@@ -12,6 +12,9 @@ namespace NuGet.Build.Tasks
     /// <typeparam name="T">The type of object to be added to the queue.</typeparam>
     public abstract class LoggingQueue<T> : IDisposable
     {
+        // To detect redundant calls
```

Comment:
nit: I don't think we need a comment here. The disposed bool is self descriptive.

Reply:
Feedback applies all throughout.